src/lexer.l

```diff
@@ -130,7 +130,7 @@ struct lexer_param;
 \.[a-zA-Z_][a-zA-Z_0-9]*  { yylval->literal = jv_string(yytext+1); return FIELD;}
 \$([a-zA-Z_][a-zA-Z_0-9]*::)*[a-zA-Z_][a-zA-Z_0-9]*  { yylval->literal = jv_string(yytext+1); return BINDING;}
 
-[ \n\t]+  {}
+[ \r\n\t]+  {}
```

Comment:
Possible to use a white space class like `[:space:]`? that would probably include some more white space characters like `\v` i think

Reply:
👍 have no strong opinion about it, maybe good to keep what is considered whitespace between jq filter and JSON in sync?